tests/30rooms/04messages.pl

```diff
@@ -215,6 +215,15 @@
             body => "Message number $_[0]"
          )
       } foreach => [ 1 .. 20 ] )->then( sub {
+         await_sync( $user, check => sub {
+            sync_timeline_contains( $_[0], $room_id, sub {
```

Comment:
You could probably use `await_sync_timeline_contains` instead of these two methods, right?

Otherwise looks fine to me.